terraform/aws/security/aws-opensearchserverless-encrypted-with-cmk.yaml

```
@@ -0,0 +1,45 @@
+rules:
+  - id: aws-opensearchserverless-encrypted-with-cmk
+    patterns:
+      - pattern-inside: |
+          resource "aws_opensearchserverless_security_policy" $ANYTHING {
+            ...
+            type =  "encryption"
+            ...
+          }
+      - pattern-either:
+          - patterns:
+              - pattern: policy = "$JSONPOLICY"
+              - metavariable-pattern:
+                  metavariable: $JSONPOLICY
+                  language: json
+                  pattern: |
+                    {..., "AWSOwnedKey":true, ... }
+          - patterns:
+              - pattern-inside: policy = jsonencode(...)
+              - pattern: |
+                  {..., AWSOwnedKey = true, ...}
+    message: Ensure opensearch serverless is encrypted at rest using AWS KMS (Key
+      Management Service)  CMK (Customer Managed Keys). CMKs give you control
```

```suggestion
      Management Service) CMK (Customer Managed Keys). CMKs give you control
```
this will just get rid of the extra space.